src/nb_ndarray.cpp

```diff
@@ -550,6 +554,8 @@ ndarray_handle *ndarray_import(PyObject *o, const ndarray_config *c,
                 converted = handle(o).attr("astype")(dtype);
             }
         } catch (...) { converted.reset(); }
+        */
+        abort();
```

Comment:
If you add an `abort`, add it before and we don't necessarily need the comment.

Reply:
This specific abort has been reworked and disappeared